app/idValidation.go

```diff
@@ -0,0 +1,14 @@
+package app
+
+import "strconv"
+
+func ValidateID(index string) int {
+
+	wantedIndex, _ := strconv.Atoi(index)
+
+	if wantedIndex < 1 || wantedIndex > 151 {
+		panic("Please introduce a valid pokemon ID from first gen. (1-151)")
```

Comment:
Please avoid panics like the plague. Remember that you can return multiple values in functions, if something can go  wrong you should return an error as well. Your function signature then would look like `func ValidateID(index string) (int, error)`.